Dockerfile

```diff
@@ -44,21 +51,35 @@ RUN \
   dovecot-core dovecot-imapd dovecot-ldap dovecot-lmtpd \
   dovecot-managesieved dovecot-pop3d dovecot-sieve dovecot-solr \
   dumb-init \
-  # E - O
-  ed fetchmail file gamin gnupg gzip iproute2 iptables \
-  locales logwatch lhasa libdate-manip-perl liblz4-tool \
+  # # E - O
+  ed fetchmail file gamin gnupg gzip gpg=2.2.12-1+deb10u1 \
+  gpg-agent=2.2.12-1+deb10u1 inotify-tools \
+  iptables locales logwatch lhasa libdate-manip-perl liblz4-tool \
   libmail-spf-perl libnet-dns-perl libsasl2-modules lrzip lzop \
   netcat-openbsd nomarch opendkim opendkim-tools opendmarc \
   # P - Z
   pax pflogsumm postgrey p7zip-full postfix-ldap postfix-pcre \
   postfix-policyd-spf-python postsrsd pyzor \
   razor rpm2cpio rsyslog sasl2-bin spamassassin supervisor \
-  unrar-free unzip whois xz-utils >/dev/null && \
+  unrar-free unzip wget=1.20.1-1.1 whois xz-utils >/dev/null && \
```

Comment:
HADOLINT reports error to use curl or wget in a Dockerfile but not both, we should switch to curl

Reply:
> HADOLINT reports error to use curl or wget in a Dockerfile but not both, we should switch to curl

Yes, we should use `curl`

Reply:
Added with https://github.com/docker-mailserver/docker-mailserver/commit/3b56e841e64b0081c62c27248a1378ba235b6160